PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
 <!--
 Thanks for contributing a pull request! Please ensure you have taken a look at
-the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#Contributing-Pull-Requests
+the contribution guidelines: http://scikit-learn.org/dev/developers/contributing.html#contributing-pull-requests
 -->
-#### Reference Issue
-<!-- Example: Fixes #1234 -->
+#### Reference Issues/PRs
+<!-- Example: Fixes #1234. Closes #2345. See also #3456. -->
```

Comment:
I think people are more likely to read what is the template than in the doc. I would add a short sentence mentioning that this is important to automatically close the issue/PR when the PR is merged and maybe add the link to https://github.com/blog/1506-closing-issues-via-pull-requests. Proposal (feel free to reword and improve it):

```
<! -- Example: Fixes #1234. Closes #2345. See also #3456. 
#1234 and #2345 will be closed when this PR is merged. A link will be added to #3456. This is important to keep our tracker organised. More details at: https://github.com/blog/1506-closing-issues-via-pull-requests
-->